templates/site/index.html

```diff
@@ -0,0 +1,209 @@
+    {% endwith %}
+
+<!--Page content area-->
+<div class="row">
+    <div class="container-fluid content-search-main">
+        <div class="content-search-left">
+        <div id="backButtonContainer" class="div-back-button-container">
+            <i class="fa-solid fa-arrow-left back-button-arrow"></i><a id="backButton" class="button-back-button subtitle-1" role="button">Home</a><img  src="images/open-in-app.svg">
+        </div>
+
+            {% with %}
+                {% set dropdown_type = "regular_dropdown_source" %}
+                {% set dropdown_button_id = "dropdownSourceRegular" %}
+                {% set dropdown_label_text = "Source" %}
+                {% include "partials/dropdown-panel.html" %}
+            {% endwith %}
+
+            {% with %}
+                {% set dropdown_type = "regular_dropdown_share" %}
+                {% set dropdown_button_id = "dropdownShareRegular" %}
+                {% set dropdown_label_text = "Share" %}
+                {% include "partials/dropdown-panel.html" %}
+            {% endwith %}
+
+        <div id="advancedFilters" class="div-advanced-filters">
+            <div>
+                <h6 class="title-advanced-filters">Advanced Filters</h6>
+            </div>
+
+            {% with %}
+                {% set dropdown_type = "dropdown_time_frame" %}
+                {% set dropdown_timeframe_id = "dropdownTimeframe" %}
+                {% set dropdown_timeframe_link_id = "dropdownTimeframeLabel" %}
+                {% set dropdown_timeframe_list_id = "dropdownTimeFrameList" %}
+                {% set user_datepicker_start_id  = "timeframeStartDatePicker" %}
+                {% set user_timepicker_start_id = "timeframeStartTimePicker" %}
+                {% set user_datepicker_end_id  = "timeframeEndDatePicker" %}
+                {% set user_timepicker_end_id = "timeframeEndTimePicker" %}
+                {% set user_checkbox_allday_id = "timeframeCheckboxAllDay" %}
+                {% set user_dropdown_repeat_id = "timeframeDropdownRepeat" %}
+                {% set user_dropdown_repeat_label_id = "timeframeDropdownRepeatLabel" %}
+                {% set user_datepicker_endson_id = "timeframeEndsOnDatepicker" %}
+                {% set dropdown_label_text = "Time Frame" %}
+                {% include "partials/dropdown-panel.html" %}
+            {% endwith %}
+
+            <div>
+                {% with %}
+                    {% set button_type = "button_location" %}
+                    {% set button_id = "showLocationMap" %}
+                    {% set button_label = "Location" %}
+                    {% include "partials/button.html" %}
+                {% endwith %}
+            </div>
+
+            {% with %}
+                {% include "partials/modal-map.html" %}
+            {% endwith %}
+
+            <div id="nonEnumQueryables" class="div-queryable-filters"></div>
+
+            <div id="dropdownQueryables" class="div-queryable-filters"></div>
```

Comment:
Why are some labels left aligned and some centred?
Checkbox should be vertically aligned to the centre of the label. 

![image](https://github.com/user-attachments/assets/7ef7c588-9648-4542-93d0-9eac234aa022)